ldms/src/sampler/spank/slurm_notifier.c

```diff
@@ -74,9 +74,11 @@
 static char *stream;
 #define SLURM_NOTIFY_TIMEOUT 5
 static time_t io_timeout = SLURM_NOTIFY_TIMEOUT;
+static int user_debug = 0;
 
 static const char *stepd_event = "";
 #define DEBUG2(FMT, ...) do { \
+	if (user_debug) \
```

Comment:
This needs an else,
```
slurm_debug2(...)
```
otherwise without _debug_ we get no messages at all regardless of the slurm message level configuration.

Reply:
@baallan, That seems reasonable to me.